connector-common-http/src/main/java/com/redhat/cloud/notifications/connector/http/HttpExceptionProcessor.java

```diff
@@ -49,7 +50,7 @@ protected void process(Throwable t, Exchange exchange) {
             } else {
                 logHttpError(ERROR, e, exchange);
             }
-        } else if (t instanceof ConnectTimeoutException) {
+        } else if (t instanceof ConnectTimeoutException || t instanceof SocketTimeoutException) {
```

Comment:
SocketTimeout happens when the connection has been established already and we're waiting too long for data. It's different from a connection timeout so introducing a new error type would make sense.

Reply:
I'm not sure I'm clear so here's an additional comment: I would process the SocketTimeoutException in a new `else if (t instanceof SocketTimeoutException` block and use a new `HttpErrorType` value dedicated to that case.